rasa_core/policies/mapping_policy.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os
+from typing import Any, List, Text
+
+from rasa_core.actions.action import ACTION_LISTEN_NAME
+
+from rasa_core import utils
+from rasa_core.domain import Domain
+from rasa_core.policies.policy import Policy
+from rasa_core.trackers import DialogueStateTracker
+from rasa_core.constants import MAPPING_SCORE
+
+logger = logging.getLogger(__name__)
+
+
+class MappingPolicy(Policy):
```

Comment:
should this by default map `restart` intent to `ActionRestart`? I'd love to get this handled properly as part of this